blog.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/stylesheet.css">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
+
+      <nav>
+        <ul>
+          <li class="navlist"><a href="index.html">Home</a></li>
+          <li class="navlist"><a href="dogs.html">Dogs</a></li>
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+      <h1>Adoptadog Blog</h1>
+
+      <div class="blog-page">
+
+          <figure class="blog-image">
+            <img src="images/blog-1.jpg" alt="Dog and owner hiking">
+          </figure>
+
+          <article class="blog">
+
+            <h2>Traveling With Your Dog</h2>
+
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+            <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+          </article>
+      </div>
+
+      <div class="blog-page">
+
+          <figure class="blog-image">
```

Comment:
Would suggest you use flexbox for your blog-page css. So when the screen size shrinks, your text will go below the image. Right now it just gets squished together